br/pkg/restore/data.go

```diff
@@ -242,16 +268,16 @@ func (recovery *Recovery) dumpRegionInfo() {
 func (recovery *Recovery) dumpRecoverPlan() {
 	log.Debug("dump recovery plan")
 	// Group region peer info by region id.
-	for storeID, _ := range recovery.recoveryPlan {
-		log.Debug("TiKV", zap.Uint64("store id", storeID))
+	for storeId := range recovery.recoveryPlan {
+		log.Debug("TiKV", zap.Uint64("store id", storeId))
 	}
 }
 
 // send the recovery plan to recovery region (force leader etc)
 func (recovery *Recovery) RecoverRegions(ctx context.Context, allStores []*metapb.Store) (err error) {
 	eg, ectx := errgroup.WithContext(ctx)
-	totalTiKVs := len(recovery.recoveryPlan)
-	workers := utils.NewWorkerPool(uint(totalTiKVs), "Recover Regions")
+	totalStores := len(recovery.recoveryPlan)
```

Comment:
```suggestion
	totalStores := recovery.totalStores
```

Reply:
It is not that all tikv have its recovery plan, so it should only recover the tikv that have region to recover. however, I see the name totalStores may confuse people. will change it.